mappings/net/minecraft/block/Material.mapping

```diff
@@ -11,7 +11,7 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
-	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
+	FIELD field_15921 MOSS Lnet/minecraft/class_3614;
 		COMMENT Not in use, but has foliage color.
```

Comment:
If you don't mind, consider update this javadoc as well

Reply:
ah didnt notice that thanks for the bump

Reply:
```suggestion
		COMMENT Material for the moss block. Has foliage color.
```